### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -12614,6 +12614,8 @@ sais->says
 salery->salary
 samll->small
 samller->smaller
+samwich->sandwich
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the plural version too please.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -12622,8 +12624,11 @@ santize->sanitize
 santized->sanitized
 santizes->sanitizes
 santizing->sanitizing
+sanwich->sandwich
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the plural please.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -12622,8 +12624,11 @@ santize->sanitize
...
+saphire->sapphire
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Plural

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -12649,14 +12654,19 @@ satrical->satirical
 satrically->satirically
 sattelite->satellite
 sattelites->satellites
+sattellite->satellite
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Plural

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -12622,8 +12624,11 @@ santize->sanitize
 santized->sanitized
 santizes->sanitizes
 santizing->sanitizing
+sanwich->sandwich
+saphire->sapphire
 sargant->sergeant
 sargeant->sergeant
+sargent->sergeant
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Added

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -12649,14 +12654,19 @@ satrical->satirical
 satrically->satirically
 sattelite->satellite
 sattelites->satellites
+sattellite->satellite
 saught->sought
 sav->save
 savable->saveable
 savees->saves
 saveing->saving
+savelt->svelte
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -12649,14 +12654,19 @@ satrical->satirical
...
 savely->safely
+savere->severe
 savety->safety
 savgroup->savegroup
+savve->salve
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 7 above)
[the remark this answers is not in the document]
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Sorry, I think save should be first please @Gelma .

[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

[the remark this answers is not in the document]
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Sorry, I think it should be `save, savvy, salve` in terms of likelihood @Gelma .

[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Pull request merged**